[skyr-url] CMake targets aren't exported for debug builds

**Describe the bug**
Example code of `skyr-url` can not be successfully built in debug mode with `vcpkg`. Release builds are fine, but the debug one fails because of only release library version is available to CMake, and, it leads to MSVCRT link type conflict. For some reasons `skyr-url-targets-release.cmake` is created without a debug version and picked instead of it. 

**Environment**
- OS: Windows 10 Pro
- Compiler: VS 16.11.9 (MSVC v142)
- CMake: 3.21.4
- Ninja 1.10.2

**To Reproduce**
Example instructions and code from [skyr-url](https://github.com/glynos/url) homepage.

**Expected behavior**
Successful build and run of an example.

**Failure logs** 
```
> [main] Building folder: skyr-test 
> [build] Starting build
> [proc] Executing command: D:\Dev\cmake\bin\cmake.EXE --build d:/Projects/skyr-test/build --config Debug --target all -j 18 --
> [build] [1/2  50% :: 2.288] Building CXX object CMakeFiles\url_parts.dir\url_parts.cpp.obj
> [build] D:\Dev\vcpkg\installed\x64-windows\include\range/v3/distance.hpp(20) : Warning: This header is deprecated. Please #include <range/v3/iterator/operations.hpp> instead.
> [build] [2/2 100% :: 2.414] Linking CXX executable url_parts.exe
> [build] FAILED: url_parts.exe 
> [build] cmd.exe /C "cd . && D:\Dev\cmake\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\url_parts.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\link.exe /nologo CMakeFiles\url_parts.dir\url_parts.cpp.obj  /out:url_parts.exe /implib:url_parts.lib /pdb:url_parts.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console  D:\Dev\vcpkg\installed\x64-windows\lib\skyr-url.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cmd.exe /C "cd /D D:\Projects\skyr-test\build && C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -noprofile -executionpolicy Bypass -file D:/Dev/vcpkg/scripts/buildsystems/msbuild/applocal.ps1 -targetBinary D:/Projects/skyr-test/build/url_parts.exe -installedDir D:/Dev/vcpkg/installed/x64-windows/debug/bin -OutVariable out""
> [build] LINK Pass 1: command "C:\PROGRA~2\MICROS~2\2019\COMMUN~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\link.exe /nologo CMakeFiles\url_parts.dir\url_parts.cpp.obj /out:url_parts.exe /implib:url_parts.lib /pdb:url_parts.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console D:\Dev\vcpkg\installed\x64-windows\lib\skyr-url.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\url_parts.dir/intermediate.manifest CMakeFiles\url_parts.dir/manifest.res" failed (exit code 1319) with the following output:
> [build] skyr-url.lib(ipv4_address.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(ipv4_address.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] skyr-url.lib(ipv6_address.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(ipv6_address.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] skyr-url.lib(url_search_parameters.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(url_search_parameters.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] skyr-url.lib(url.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(url.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] skyr-url.lib(parse.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(parse.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] skyr-url.lib(url_parser_context.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(url_parser_context.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] skyr-url.lib(idna.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in url_parts.cpp.obj
> [build] skyr-url.lib(idna.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in url_parts.cpp.obj
> [build] LINK : warning LNK4098: defaultlib 'MSVCRT' conflicts with use of other libs; use /NODEFAULTLIB:library
> [build] url_parts.exe : fatal error LNK1319: 14 mismatches detected
> [build] ninja: build stopped: subcommand failed.
> [build] Build finished with exit code 1
```

**Additional context**
I tried to add missing `skyr-url-targets-debug.cmake` manually (just copied `skyr-url-targets-release.cmake` and replaced IMPORTED_LOCATION_RELEASE with DEBUG) and it worked.

--- a/ports/skyr-url/portfile.cmake
+++ b/ports/skyr-url/portfile.cmake
@@ -8,9 +8,8 @@ vcpkg_from_github(
         HEAD_REF main
 )
 
-vcpkg_configure_cmake(
-        SOURCE_PATH ${SOURCE_PATH}
-        PREFER_NINJA
+vcpkg_cmake_configure(
+        SOURCE_PATH "${SOURCE_PATH}"
         OPTIONS
             -Dskyr_BUILD_TESTS=OFF
             -Dskyr_BUILD_DOCS=OFF
@@ -18,11 +17,13 @@ vcpkg_configure_cmake(
             -Dskyr_WARNINGS_AS_ERRORS=OFF
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 vcpkg_copy_pdbs()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH share/cmake/skyr-url)
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE_1_0.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE_1_0.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/skyr-url/vcpkg.json
+++ b/ports/skyr-url/vcpkg.json
@@ -1,12 +1,20 @@
 {
   "name": "skyr-url",
-  "version-string": "1.13.0",
-  "port-version": 1,
+  "version": "1.13.0",
+  "port-version": 2,
   "description": "A C++ library that implements the WhatWG URL specification",
   "homepage": "https://github.com/cpp-netlib/url",
   "dependencies": [
     "nlohmann-json",
     "range-v3",
-    "tl-expected"
+    "tl-expected",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
   ]
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6406,7 +6406,7 @@
     },
     "skyr-url": {
       "baseline": "1.13.0",
-      "port-version": 1
+      "port-version": 2
     },
     "sleef": {
       "baseline": "3.5.1",

--- a/versions/s-/skyr-url.json
+++ b/versions/s-/skyr-url.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "429bf1616eeeb9c2315b09d9a6c741245e3eae6a",
+      "version": "1.13.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "174f5bc4cc8080f2cb34c7a1aad9b14bedaa2c9a",
       "version-string": "1.13.0",
       "port-version": 1

